testing/tests/DevExpress.ui.widgets/mapParts/googleStaticTests.js

```diff
@@ -471,3 +471,24 @@ QUnit.test("click", function(assert) {
         assert.equal(eventFired, 1);
     });
 });
+
+QUnit.test("StopPropagation is not called on the pointer down event", function(assert) {
```

Comment:
Tests in this suite have a slightly different naming. Use the following scheme: "the pointer down event propagation should not be canceled"